frontend/src/App.vue

```diff
@@ -165,10 +165,16 @@ export default {
     isNoGenerators() {
       const promptName = _.get(this.currentPrompt, "name");
       const message = _.get(this.messages, "select_generator_name", "");
-      const noChoices = _.isEmpty(
-        _.get(this.currentPrompt, "questions[0].choices")
-      );
-      return promptName === message && noChoices;
+      if (promptName && promptName === message) {
```

Comment:
Try this code:

         
         
        if (promptName === message) {
		const questions = _.compact(_.get(this.currentPrompt, "questions"));
                const generatorQuestion = _.find(questions, question => {
                      return question.name === "generator" && _.isEmpty(question.choices);
               });
        
               return !_.isNil(generatorQuestion);
         }
         return false;



In case that questions are never undefined you can remove _.compact()

Reply:
ok